jib-core/src/test/java/com/google/cloud/tools/jib/builder/steps/PullBaseImageStepTest.java

```diff
@@ -127,4 +138,80 @@ public void testCall_offlineMode_cached()
 
     Mockito.verify(buildContext, Mockito.never()).newBaseImageRegistryClientFactory();
   }
+
```

Comment:
i'm skeptical about my implementation for this testing method .

Basically its 

1.  Pass in a mock manifest list to the `testObtainPlatformSpecificImageManifest` method.
2.  Method searches for the digest matching the input platform
3.  Method calls registry client with with the obtained digest to get the mock manifest

Looking for feedback

Reply:
Yeah, think about what you are testing. Most of the assertions that verifies the manifest content is unnecessary, because what you are testing is whether the method can find the digest to pass to `registryClient.pullManifest()` and whether it returns the same manifest returned by `registryClient`. `registryClient.pullManifest` doesn't even need to return an actual manifest; a mock will work. I can talk about this later.